calculator.rb

```diff
@@ -0,0 +1,119 @@
+=begin INSTRUCTIONS. Ada Calculator, Due August 8, 2018
+The program should have support for these four operations
+addition, subtraction, multiplication, and division.
+The program should accept both the name (add) and the symbol (+) for
+each possible operation. The program should handle unexpected user input
+=end
+
+# define some methods to do the math
+def addition(first_num, second_num)
+  return first_num + second_num
+end
+
+#write an addition test invoking the method in the test
+def addition_test
+  if addition(2, 3) != 5 # 2 + 3 = 5
+    raise "something is wrong with your addition method"
```

Comment:
I like that you've included code to test your code! We'll discuss specific tools and techniques for doing this next week.